udisks/mount_options.conf.in

```diff
@@ -0,0 +1,26 @@
+# This file contains custom mount options for udisks 2.x
+# Typically placed at /etc/udisks2/mount_options.conf
+# Refer to http://storaged.org/doc/udisks2-api/latest/mount_options.html
+#
+
+### Simple global overrides
+# [defaults]
+# # common options, applied to any filesystem, always merged with specific filesystem type options
+# defaults=ro
+# allow=exec,noexec,nodev,nosuid,atime,noatime,nodiratime,ro,rw,sync,dirsync,noload
+
+### Specific filesystem type options
+# vfat_defaults=uid=,gid=,shortname=mixed,utf8=1,showexec,flush
```

Comment:
The example should also use the `uid=$UID` and `gid=$GID` notation.

Reply:
Ahh, totally forgot about that. Fixed.